tests/testthat/test-guides.R

```diff
@@ -601,10 +601,10 @@ test_that("guides title and text are positioned correctly", {
       scale_fill_continuous(name = "the\ncontinuous\ncolorscale")
   )
   expect_doppelganger("vertical gap of 1cm between guide title and guide",
-    p + theme(legend.spacing.y = grid::unit(1, "cm"))
+    p + theme(legend.title = element_text(margin = margin(b = 1, unit = "cm")))
```

Comment:
I figured out why the spacing between guides has changed. Here, the legend spacing is default and no longer 1 cm. So, it is actually doing what it is supposed to do and this isn't a regression.